x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/saml/SamlObjectHandler.java

```diff
@@ -322,4 +331,74 @@ protected void validateNotOnOrAfter(DateTime notOnOrAfter) {
             throw samlException("Rejecting SAML assertion because [{}] is on/after [{}]", pastNow, notOnOrAfter);
         }
     }
+
+    protected ParsedQueryString parseQueryStringAndValidateSignature(String queryString, String samlMessageParameterName) {
```

Comment:
Again, assuming that you didn't change much when you copied this over ( apart from introducing `samlMessageParameterName` ) , please do flag if otherwise ! ( :octocat: diffs are not that helpful in spotting this kind of things :) )

Reply:
Good spot! Indeed this is the only significant change as part of the refactoring. Next time I'll compile the list of changes before hand to save you some time. Sorry about that. Here is a list for rest of the changes (they are all very minor):
* method visibility from `private` to `protect` due to superclass extraction. 
  - This includes: `SamlResponseHandler#isSuccess`, `SamlResponseHandler#getStatusCodeMessage`, `SamlObjectHandler#decodeBase64`, `SamlObjectHandler#inflate`, and `SamlObjectHandler#parseQueryStringAndValidateSignature`.
* Reverse the order of String equals compare in `SamlResponseHandler#isSuccess`. 
  - Changed from `status.getStatusCode().getValue().equals(StatusCode.SUCCESS)` to `StatusCode.SUCCESS.equals(status.getStatusCode().getValue())`.